app/controllers/catalog_controller.rb

```diff
@@ -323,11 +323,7 @@ class CatalogController < ApplicationController
                                                         options[:document] &&
                                                           !options[:document].restricted?
                                                       }
-    config.add_show_tools_partial :searchworks_url, component: Earthworks::SearchworksUrl,
```

Comment:
Can we remove the SearchworksUrl now? I don't think anything else calls it.

Reply:
Yes, that component was removed as part of this PR.

Reply:
Oops. @jcoyne let us know if this looks OK to you!